src/styles/variables.css

```diff
@@ -43,6 +44,9 @@
   --temp-grey-blue-6: rgb(231, 242, 246); /* Used for user profile gradient end color */
   --temp-grey-blue-7: rgb(118, 116, 139); /* Used for the wallet connect button in the user navigation */
   --drop-shadow: rgba(62, 118, 244, 0.14);
+  --light-border: rgba(54, 54, 54, 0.15);
+  --text-hover: rgb(76,93,255);
+  --text-disabled: rgb(194,204,204);
```

Comment:
Two things here:
- the names for these variables is very generic, please come up with a better name
- if the color is only used in one place, don't add it to the global variables, add it locally in the component's css module, using the `@value` declaration